lib/src/cupertino/field/cupertino_typeahead_form_field.dart

```diff
@@ -43,7 +43,8 @@ class CupertinoTypeAheadFormField<T> extends FormField<String> {
     required SuggestionSelectionCallback<T> onSuggestionSelected,
     required ItemBuilder<T> itemBuilder,
     IndexedWidgetBuilder? itemSeparatorBuilder,
-    required SuggestionsCallback<T> suggestionsCallback,
+    SuggestionsCallback<T>? suggestionsCallback,
+    SuggestionsLoadMoreCallback<T>? suggestionsLoadMoreCallback,
```

Comment:
an idea for making this more convenient could also be:
a separate constructor for the load more mechanism, so that the callbacks can always be `required`.
This would also be non-breaking and additionally would ensure we dont have to run asserts or forget the callback altogether.

It would mean we have to write two full constructors, however. Not sure if we want to do that. It would be nicer from a user perspective though.

The constructor could be something like `TypeAheadFormField.paged`.

Reply:
Ok, Will create two new full constructor , One is Unnamed constructor, another one is named as `TypeAheadFormField.paged`